homeassistant/components/sensor/serial_pm.py

```diff
@@ -21,7 +21,6 @@
 CONF_BRAND = "brand"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_PLATFORM): 'serial_pm',
```

Comment:
I'm confused, why doesn't this work ? It should work. `platform` should always be set to `serial_pm`, or else this platform would have never been loaded.